library/std/src/sys/pal/vexos/thread.rs

```diff
@@ -0,0 +1,44 @@
+use super::unsupported;
+use crate::ffi::CStr;
+use crate::io;
+use crate::num::NonZero;
+use crate::time::Duration;
+
+pub struct Thread(!);
+
+pub const DEFAULT_MIN_STACK_SIZE: usize = 64 * 1024;
+
+impl Thread {
+    // unsafe: see thread::Builder::spawn_unchecked for safety requirements
+    pub unsafe fn new(_stack: usize, _p: Box<dyn FnOnce()>) -> io::Result<Thread> {
+        unsupported()
+    }
+
+    pub fn yield_now() {
+        unsafe {
+            vex_sdk::vexTasksRun();
+        }
+    }
+
+    pub fn set_name(_name: &CStr) {
+        // nope
+    }
+
+    pub fn sleep(dur: Duration) {
+        let start = Instant::now();
+
+        while start.elapsed() < dur {
+            unsafe {
+                vex_sdk::vexTasksRun();
+            }
```

Comment:
```suggestion
            Thread::yield_now();
```

Reply:
Think I prefer to be more explicit here - it's overall less indirection and easier to see what's going on in the implementation if this is kept as its own call to `vexTasksRun`.